### PR TITLE
Use title translations

### LINKED ISSUE #6605: Make use of title translations in Top Stories and Features & Analysis containers
### Is your feature request related to a problem? Please describe.
A full list of title translations already exists for Top Stories and Features & Analysis, but they aren't being used.

### Describe the solution you'd like
Ensure translations are being used in the following containers, if one exists, otherwise fallback to a default title:
- Container: `CpsFeaturesAnalysis`. Default title: "Features & Analysis"
- Container: `CpsTopStories`. Default title: "Top Stories"
- Resolve undefined default title in container: `CpsRelatedContent`. Default title: "Related Content"

#### Translation key names
```js
featuresAnalysisTitle: 'Ivyo BBC Gahuza ibahitiramwo'
topStoriesTitle: "Inkuru iri kw'isonga",
relatedContent: 'Ibindi bisa n’ibi',
```
#### Example implementation
```js
const { dir, translations } = useContext(ServiceContext);

const title = pathOr(
  'Default Title',
  ['topStoriesTitle'],
  translations,
);

return (
  <CpsOnwardJourney
    labelId="top-stories-heading"
    title={title}
    content={content}
    parentColumns={parentColumns}
    singleTransform={singleTransform}
    listTransform={listTransform}
  />
);
```

### Describe alternatives you've considered
n/a

### Testing notes
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

### Additional context
n/a

### PR DESCRIPTION
**Pull request by FK78**
Resolves #6605

**Overall change:**
_Use title translations._

**Code changes:**

- _Request the translations using `ServiceContext`._
- _Add in pathOr to extract translations and add a fallback to a default title._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [x] This PR requires manual testing

### CODE REVIEW

**Review by rhenshaw56 — APPROVED** [review]

File: src/app/containers/CpsFeaturesAnalysis/index.jsx [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
 import { ServiceContext } from '#contexts/ServiceContext';
 import StoryPromo from '../StoryPromo';
 import CpsOnwardJourney from '../CpsOnwardJourney';
 
 const FeaturesAnalysis = ({ content, parentColumns }) => {
-  const { dir } = useContext(ServiceContext);
+  const { dir, translations } = useContext(ServiceContext);
+
+  const title = pathOr(
+    'Features & Analysis',
```
Comment (andrewscfc) [inline src/app/containers/CpsFeaturesAnalysis/index.jsx:15]: Do we need the escaped & (`&amp`) in the fallback like we used to have below: `Features &amp; Analysis`?

Also we should probably add some new tests for the fallback of the `pathOr` in each container.

**Review by andrewscfc — COMMENTED** [review]
Bit of extra test coverage we could add 👍

File: src/app/containers/CpsFeaturesAnalysis/index.jsx (hunk 1 above)
Comment (andrewscfc) [reply src/app/containers/CpsFeaturesAnalysis/index.jsx:15]: (I don't know the answer to this but would be worth checking)

**Review by oluoluoxenfree — APPROVED** [review]

File: src/app/containers/CpsFeaturesAnalysis/index.jsx (hunk 1 above)
Comment (FK78) [reply src/app/containers/CpsFeaturesAnalysis/index.jsx:15]: In regards to the `&amp`, I've tested it locally and using `&` doesn't cause any issues.

**Comment by paruchurisilpa** [comment]
Looks good to me..

**Pull request merged**